tools/doxygen/doxygen.cmake

```diff
@@ -501,7 +509,12 @@ function(make_doxygen_target modules_var)
     set_target_properties("DOXY-generate-processor-previews" 
                             PROPERTIES FOLDER "doc" EXCLUDE_FROM_ALL TRUE)
 
-    add_dependencies("DOXY-Inviwo" "DOXY-generate-processor-previews" "DOXY-Clear")
+    option(IVW_DOXYGEN_OPTIONAL_PROCESSOR_PREVIEWS "Turn on this to disable generating processor previews when building doxygen documentation (DOXY-Inviwo)" OFF)
```

Comment:
Um, turn on to disable but the option is still called `IVW_DOXYGEN_OPTIONAL_PROCESSOR_PREVIEWS`? Maybe have CMake option `ON` by default or rename to `IVW_DOXYGEN_OPTIONAL_PROCESSOR_PREVIEWS_DISABLED` or similar. 
Either way, should remove `"this"` in the string.

Reply:
I agree that the wording is not the best. 
The reason for the option is to avoid to have to build the whole `inviwo.exe` when you want to build the documentation only. (`DOXY-generate-processor-previews` depends on `inviwo.exe` and if you change the documentation in a file in core, you have to wait 20 min to see the results)

Reply:
The `IVW_DOXYGEN_OPTIONAL_PROCESSOR_PREVIEWS_DISABLED` feels a little like double negation.  The optional is to there to remove the dependency to the preview generator target. The target is still there but has to be built manually when needed. Appending a disabled to the option name feels wrong. Maybe something like, enforce_processor_previews that is on by default?

Reply:
how about negating the if statement then? Then the previews would be optional.